### DISCUSSION TITLE
Single queue/ride entrance exit for möbius coasters

### DISCUSSION THREAD

**Discussion by astraylife1** [OpenRCT2/OpenRCT2#18398] [Feature requests]
Not sure how this is possible but it would be great to have a single queue for möbius coasters. I know Brian Andrelczyk has made a youtube tutorial for this but it is extremely complex. Maybe there could be an option for single entrance/exit for rides with multiple stations.

**Comment by James103** [comment]
In general, this would require a queue that splits into multiple at some point, such that one queue entrance serves multiple ride entrances.

**Reply by astraylife1** [reply]
I've tried having a single queue and using the tile inspector to make a T intersection for the queue to split. However, the peeps end up bunching up at the entrances and eventually it doesn't work as expected. Just wondering with the new save format if there would eventually be an easier way to achieve this.
